src/py/flwr/common/serde_test.py

```diff
@@ -427,6 +427,10 @@ def test_metadata_serialization_deserialization() -> None:
     maker = RecordMaker()
     original = maker.metadata()
 
+    # A Metadata obj is fully defined when it's used by a Message
+    # The Message sets the metadata's .created_at
+    _ = Message(metadata=original, content=maker.recordset(1, 0, 0))
```

Comment:
A `Metadata` object is not fully defined if it's not used to construct a `Message`. that's because the `Message` constructor sets the metadata`.created_at`. Because of this reason, this test is identical to the next one so I vote for its removal. @chongshenng @danieljanes